src/dtos/http_request.go

```diff
@@ -0,0 +1,15 @@
+package dtos
+
+import "github.com/Real-Dev-Squad/wisee-backend/src/models"
+
+type CreateUpdateFormRequestDto struct {
+	Status        *string            `json:"status"`
+	Content       models.FormContent `json:"content"`
+	PerformedById int64              `json:"performed_by_id"`
```

Comment:
- TODO:
 - [ ] Remove the `PerformedById` and `ResponseById` once auth is integrated